chassis/core/src/main/java/com/griddynamics/jagger/master/MasterToJaasCoordinator.java

```diff
@@ -205,22 +209,39 @@ public StatusExchangeThread(String sessionCookie) {
         }
         
         public void setPendingRequestEntity() {
+            while (standBy && lastStatus == TestEnvironmentStatus.RUNNING && !runningStatusSent) {
+                try {
+                    TimeUnit.MILLISECONDS.sleep(100);
+                } catch (InterruptedException e) {
+                    LOGGER.error("Error occurred while sleeping", e);
+                    standBy = false;
+                }
+            }
```

Comment:
It prevents master from sending pending status until running status is sent.  Since, status is sent by another thread every 15 sec, the only way to do it is to wait until running status is sent and then update response entity in that thread.